website/docs/rbi.md

```diff
@@ -196,6 +196,28 @@ RBI files in an `rbi/` directory. When they do this, Tapioca will automatically
 merge those definitions in the autogenerated RBIs. In the future, we anticipate
 this to be the preferred way to include RBI files into a project.
 
+## Versioning for standard library RBIs
+
+There is no versioning for gems whose RBIs live in Sorbet itself. Instead,
+Sorbet tries to provide a best effort one-size-fits-all set of RBIs.
+
+Consider some examples:
+
+- A new version of Ruby **adds** a method to a class defined in the standard
+  library.
+
+  We will accept PRs to add an RBI definition for the new method. This means
+  that Sorbet **will not** report "method does not exist errors" for codebases
+  that are still on the old version of Ruby.
+
+- A new version of Ruby **deletes** a method that was deprecated.
+
+  We **will not** accept PRs to remove the corresponding method definition from
+  Sorbet, as that would prevent users on old Sorbet versions from calling it.
```

Comment:
Wouldn't it be okay to remove methods that have been removed from the minimum Ruby version Sorbet requires?

Reply:
Possibly, though this has never come up yet.